scripts/extensions/sams/src/ui/list/index.ts

```diff
@@ -0,0 +1,8 @@
+export {ListHeader} from './ListHeader';
```

Comment:
Is this proxy file useful for anything? When I want to import something, I write the name and the editor autocompletes and auto imports it no matter where it is.

And if I didn't have auto-import it'd be annoying to navigate such empty proxy files just to get to the definition

Reply:
It allows the developer to write
```js
import {ListItem, ListItemActionMenu, ListItemBorder, ListItemColumn, ListItemRow} from '../../ui/list';
```
instead of
```js
import {ListItem} from '../../ui/list/ListItem';
import {ListItemActionMenu} from '../../ui/list/ListItemActionMenu';
import {ListItemBorder} from '../../ui/list/ListItemBorder';
import {ListItemColumn} from '../../ui/list/ListItemColumn';
import {ListItemRow} from '../../ui/list/ListItemRow';
```

@fritzSF / @pavlovicnemanja As this part of the code will eventually be destined for the `ui-framework` library, which would you prefer? Or any other suggestions